franco/Project/org.fundacionjala.automationproject/src/test/resources/features/registeruser.feature

```diff
@@ -0,0 +1,26 @@
+@RegisterUser
+Feature: RegisterUser
+  Test successfully registration of user
+
+  Background:
+    Given I navigate to Main Page
```

Comment:
A scenario with two Given ????
I think that you could user PicoContainer and simplify work.

Reply:
I refactored the first given to:
Given I navigate to Main Page
and I refactored the second Given to:
Given I select Registration option in Menu